File: workspace-servidor/flotix2021/src/main/java/com/flotix/dto/VehiculoDTO.java
package com.flotix.dto;

/**
 * Objeto que almacena la informacion de los vehiculos para su devolucion
 * 
 * @author Flor
 *
 */
public class VehiculoDTO {

	private String id;
	private String matricula;
	private String fechaMatriculacion;
	private String modelo;
	private int plazas;
	private int capacidad;
	private int km;
	private boolean disponibilidad;
	private boolean baja;
	private String nombreImagen;
	private String nombreImagenPermiso;
	private String mantenimientoNoCreado;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getMatricula() {
		return matricula;
	}

	public void setMatricula(String matricula) {
		this.matricula = matricula;
	}

	public String getFechaMatriculacion() {
		return fechaMatriculacion;
	}

	public void setFechaMatriculacion(String fechaMatriculacion) {
		this.fechaMatriculacion = fechaMatriculacion;
	}

	public String getModelo() {
		return modelo;
	}

	public void setModelo(String modelo) {
		this.modelo = modelo;
	}

	public int getPlazas() {
		return plazas;
	}

	public void setPlazas(int plazas) {
		this.plazas = plazas;
	}

	public int getCapacidad() {
		return capacidad;
	}

	public void setCapacidad(int capacidad) {
		this.capacidad = capacidad;
	}

	public int getKm() {
		return km;
	}

	public void setKm(int km) {
		this.km = km;
	}

	public boolean isDisponibilidad() {
		return disponibilidad;
	}

	public void setDisponibilidad(boolean disponibilidad) {
		this.disponibilidad = disponibilidad;
	}

	public boolean isBaja() {
		return baja;
	}

	public void setBaja(boolean baja) {
		this.baja = baja;
	}

	public String getNombreImagen() {
		return nombreImagen;
	}

	public void setNombreImagen(String nombreImagen) {
		this.nombreImagen = nombreImagen;
	}

	public String getNombreImagenPermiso() {
		return nombreImagenPermiso;
	}

	public void setNombreImagenPermiso(String nombreImagenPermiso) {
		this.nombreImagenPermiso = nombreImagenPermiso;
	}

	public String getMantenimientoNoCreado() {
		return mantenimientoNoCreado;
	}

	public void setMantenimientoNoCreado(String mantenimientoNoCreado) {
		this.mantenimientoNoCreado = mantenimientoNoCreado;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelDTO/VehiculoDTO.cs

namespace Flotix2021.ModelDTO
{
    public class VehiculoDTO
    {
        private string _id;

        public string id
        {
            get { return _id; }
            set { _id = value; }
        }

        private string _matricula;

        public string matricula
        {
            get { return _matricula; }
            set { _matricula = value; }
        }

        private string _fechaMatriculacion;

        public string fechaMatriculacion
        {
            get { return _fechaMatriculacion; }
            set { _fechaMatriculacion = value; }
        }

        private string _modelo;

        public string modelo
        {
            get { return _modelo; }
            set { _modelo = value; }
        }

        private int _plazas;

        public int plazas
        {
            get { return _plazas; }
            set { _plazas = value; }
        }

        private int _capacidad;

        public int capacidad
        {
            get { return _capacidad; }
            set { _capacidad = value; }
        }

        private int _km;

        public int km
        {
            get { return _km; }
            set { _km = value; }
        }

        private bool _disponibilidad;

        public bool disponibilidad
        {
            get { return _disponibilidad; }
            set { _disponibilidad = value; }
        }

        private bool _baja;

        public bool baja
        {
            get { return _baja; }
            set { _baja = value; }
        }

    }
}
